main.py

```diff
@@ -0,0 +1,50 @@
+from whetstone import Whetstone
+import json
+from os import path, makedirs
+
+
+def extract(api, endpoint, tag=False):
+    filename = f"data/{endpoint}.json"
+    if path.exists(filename):
```

Comment:
This is probably easier to read as `if not path.exists(filename):` so you don't need an `else`.

Reply:
Makes sense.